File: todo-client/src/app/component/todo-list/Todo.ts
export class Todo {
  id: number
  title: string
  targetedDate: Date
  description: string
  iscompleted: boolean
  constructor(id: number, title: string, targetedDate: Date, description: string, iscompleted: boolean) {
    this.id = id
    this.title = title
    this.targetedDate = targetedDate
    this.description = description
    this.iscompleted = iscompleted
  }
}

File: todo-service/src/main/java/com/sio/todoservice/model/Todo.java
package com.sio.todoservice.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Date;

@NoArgsConstructor
@Getter
@Setter
@AllArgsConstructor
public class Todo {
    private int id;
    private String title;
    private Date targetedDate;
    private String description;
    private boolean isCompleted;
}
